### PR TITLE
Adds a better warning to delete-all.

### PR DESCRIPTION
**Pull request by jdoliner**

### CODE REVIEW

**Comment by JoeyZwicker** [comment]
I like this!

**Comment by jdoliner** [comment]
@JoeyZwicker I thought you might :sunglasses: Did you notice the part where it prints it in scary red so we'll be extra careful not to delete stuff we shouldn't?

**Comment by jdoliner** [comment]
screenshot from 2017-12-15 16-42-38

File: src/server/pps/cmds/cmds.go [hunk 1]
```diff
@@ -505,7 +505,7 @@ All jobs created by a pipeline will create commits in the pipeline's repo.
 			}
 			pipelineInfos, err := client.ListPipeline()
 			if err != nil {
-				return grpcutil.ScrubGRPC(err)
+				return err
```
Comment (sjezewski) [inline src/server/pps/cmds/cmds.go]: Why this change? It seems like the `ScrubGRPC` function should leave the error alone if it's not a grpc error w an error code. Do we want to display grpc error codes here specifically?

Comment (jdoliner) [reply src/server/pps/cmds/cmds.go]: Oh I just removed this because I realized it was redundant, `ListPipeline` is already calling `ScrubGRPC` on the error before it returns it. Maybe I should just do a larger pass and make sure all scrubbing happens in the client. It seems like the sanest option to just having the scrubbing right next to the actually GRPC calls.
